backend/Dockerfile

```diff
@@ -7,6 +7,9 @@ RUN pip install -r requirements.txt
 ADD . /
 WORKDIR /
 
-CMD python manage.py makemigrations authentication && python manage.py makemigrations eventposts && python manage.py migrate && python manage.py runserver 0.0.0.0:9000
+## POSTGRES WAIT SCRIPT FROM https://github.com/ufoscout/docker-compose-wait
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.9.0/wait /wait
+RUN chmod +x /wait
+CMD /wait && python manage.py makemigrations authentication && python manage.py makemigrations eventposts && python manage.py migrate && python manage.py test eventposts && python manage.py runserver 0.0.0.0:9000
```

Comment:
I am assuming we will remove "eventposts" from the test command here.

Reply:
Since it will be fixed by your branch, I'm leaving that line as it is for this PR. Thanks for reviewing!